vaadin-time-picker-flow-parent/vaadin-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/timepicker/tests/TimePickerIT.java

```diff
@@ -115,9 +115,11 @@ public void timePickerWithMinAndMaxSetting() {
         Assert.assertEquals(
                 "The first item in the dropdown should be the min value",
                 "5:00 AM", picker.getItemText(0));
+        // Make sure the item is in the viewport / rendered
+        picker.scrollToItem(1000);
         Assert.assertEquals(
                 "The last item in the dropdown should be the max value",
-                "4:00 PM", picker.getLastItemText());
+                "6:00 PM", picker.getLastItemText());
```

Comment:
Picked this change from the `main` branch. The max is configured as 6pm and the test was probably working because the item was not rendered previously. Could also be an issue with how `getLastItemText` works.